<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ICQInfoBase</class>
  <widget class="QWidget" name="ICQInfoBase" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>372</width>
        <height>282</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Form5</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item>
        <widget class="QTabWidget" name="tabWnd" >
          <widget class="QWidget" name="tab" >
            <attribute name="title" >
              <string>&amp;Names</string>
            </attribute>
            <layout class="QGridLayout" >
              <property name="margin" stdset="0" >
                <number>11</number>
              </property>
              <property name="spacing" stdset="0" >
                <number>6</number>
              </property>
              <item row="4" column="0" >
                <widget class="QLabel" name="lblLast" >
                  <property name="font" stdset="0" >
                    <font>
                      <family></family>
                      <pointsize>0</pointsize>
                      <weight>0</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                      <underline>false</underline>
                      <strikeout>false</strikeout>
                    </font>
                  </property>
                  <property name="text" stdset="0" >
                    <string>Last Name:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="5" column="1" >
                <widget class="QLineEdit" name="edtNick" />
              </item>
              <item row="3" column="1" >
                <widget class="QLineEdit" name="edtFirst" />
              </item>
              <item row="0" column="0" >
                <widget class="QLabel" name="lblUin" >
                  <property name="font" stdset="0" >
                    <font>
                      <family></family>
                      <pointsize>0</pointsize>
                      <weight>0</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                      <underline>false</underline>
                      <strikeout>false</strikeout>
                    </font>
                  </property>
                  <property name="text" stdset="0" >
                    <string>Uin:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="5" column="0" >
                <widget class="QLabel" name="TextLabel30" >
                  <property name="text" stdset="0" >
                    <string>Nick:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="4" column="1" >
                <widget class="QLineEdit" name="edtLast" />
              </item>
              <item rowspan="1" row="1" column="0" colspan="2" >
                <widget class="Line" name="lineDiv" >
                  <property name="sizePolicy" stdset="0" >
                    <sizepolicy>
                      <hsizetype>1</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="palette" stdset="0" >
                    <palette>
                      <active>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>192</red>
                          <green>192</green>
                          <blue>192</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>223</red>
                          <green>223</green>
                          <blue>223</blue>
                        </color>
                        <color>
                          <red>96</red>
                          <green>96</green>
                          <blue>96</blue>
                        </color>
                        <color>
                          <red>128</red>
                          <green>128</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>192</red>
                          <green>192</green>
                          <blue>192</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                      </active>
                      <inactive>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>192</red>
                          <green>192</green>
                          <blue>192</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>220</red>
                          <green>220</green>
                          <blue>220</blue>
                        </color>
                        <color>
                          <red>96</red>
                          <green>96</green>
                          <blue>96</blue>
                        </color>
                        <color>
                          <red>128</red>
                          <green>128</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>192</red>
                          <green>192</green>
                          <blue>192</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                      </inactive>
                      <disabled>
                        <color>
                          <red>128</red>
                          <green>128</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>192</red>
                          <green>192</green>
                          <blue>192</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>220</red>
                          <green>220</green>
                          <blue>220</blue>
                        </color>
                        <color>
                          <red>96</red>
                          <green>96</green>
                          <blue>96</blue>
                        </color>
                        <color>
                          <red>128</red>
                          <green>128</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>128</red>
                          <green>128</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>128</red>
                          <green>128</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                        <color>
                          <red>192</red>
                          <green>192</green>
                          <blue>192</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>0</blue>
                        </color>
                        <color>
                          <red>0</red>
                          <green>0</green>
                          <blue>128</blue>
                        </color>
                        <color>
                          <red>255</red>
                          <green>255</green>
                          <blue>255</blue>
                        </color>
                      </disabled>
                    </palette>
                  </property>
                  <property name="frameShape" stdset="0" >
                    <enum>QFrame::HLine</enum>
                  </property>
                  <property name="frameShadow" stdset="0" >
                    <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="frameShape" stdset="0" >
                    <enum>QFrame::HLine</enum>
                  </property>
                </widget>
              </item>
              <item row="0" column="1" >
                <layout class="QHBoxLayout" >
                  <property name="margin" stdset="0" >
                    <number>0</number>
                  </property>
                  <property name="spacing" stdset="0" >
                    <number>6</number>
                  </property>
                  <item>
                    <widget class="QLineEdit" name="edtUin" >
                      <property name="font" stdset="0" >
                        <font>
                          <family></family>
                          <pointsize>0</pointsize>
                          <weight>0</weight>
                          <italic>false</italic>
                          <bold>false</bold>
                          <underline>false</underline>
                          <strikeout>false</strikeout>
                        </font>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <spacer name="Spacer26" >
                      <property name="sizeHint" >
                        <size>
                          <width>20</width>
                          <height>20</height>
                        </size>
                      </property>
                      <property name="sizeType" >
                        <enum>Expanding</enum>
                      </property>
                      <property name="orientation" >
                        <enum>Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                </layout>
              </item>
              <item rowspan="2" row="2" column="0" colspan="1" >
                <widget class="QLabel" name="lblFirst" >
                  <property name="text" stdset="0" >
                    <string>First Name:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item rowspan="1" row="6" column="0" colspan="2" >
                <widget class="Line" name="Line2" >
                  <property name="sizePolicy" stdset="0" >
                    <sizepolicy>
                      <hsizetype>1</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                </widget>
              </item>
              <item row="10" column="1" >
                <spacer name="Spacer7" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
              <item rowspan="1" row="8" column="0" colspan="2" >
                <widget class="QLabel" name="lblRandom" >
                  <property name="text" stdset="0" >
                    <string>Random chat group:</string>
                  </property>
                </widget>
              </item>
              <item rowspan="1" row="9" column="0" colspan="2" >
                <widget class="QComboBox" name="cmbRandom" />
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="tab" >
            <attribute name="title" >
              <string>&amp;Status</string>
            </attribute>
            <layout class="QGridLayout" >
              <property name="margin" stdset="0" >
                <number>11</number>
              </property>
              <property name="spacing" stdset="0" >
                <number>6</number>
              </property>
              <item row="0" column="0" >
                <widget class="QLabel" name="TextLabel1_3" >
                  <property name="text" stdset="0" >
                    <string>Status:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="0" column="1" >
                <widget class="QComboBox" name="cmbStatus" />
              </item>
              <item row="1" column="0" >
                <widget class="QLabel" name="lblOnline" >
                  <property name="text" stdset="0" >
                    <string>Online:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="1" column="1" >
                <widget class="QLineEdit" name="edtOnline" />
              </item>
              <item row="2" column="0" >
                <widget class="QLabel" name="lblNA" >
                  <property name="text" stdset="0" >
                    <string/>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="2" column="1" >
                <widget class="QLineEdit" name="edtNA" >
                  <property name="font" stdset="0" >
                    <font>
                      <family></family>
                      <pointsize>0</pointsize>
                      <weight>0</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                      <underline>false</underline>
                      <strikeout>false</strikeout>
                    </font>
                  </property>
                </widget>
              </item>
              <item row="3" column="0" >
                <widget class="QLabel" name="lblExtIP" >
                  <property name="text" stdset="0" >
                    <string>External IP:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="3" column="1" >
                <widget class="QLineEdit" name="edtExtIP" />
              </item>
              <item row="4" column="0" >
                <widget class="QLabel" name="lblIntIP" >
                  <property name="text" stdset="0" >
                    <string>Internal IP:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="4" column="1" >
                <widget class="QLineEdit" name="edtIntIP" />
              </item>
              <item row="5" column="0" >
                <widget class="QLabel" name="lblClient" >
                  <property name="text" stdset="0" >
                    <string>Client:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="5" column="1" >
                <widget class="QLineEdit" name="edtClient" />
              </item>
              <item row="7" column="1" >
                <spacer name="Spacer6_2" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
              <item rowspan="1" row="6" column="0" colspan="2" >
                <widget class="Q3MultiLineEdit" name="edtAutoReply" />
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="password" >
            <attribute name="title" >
              <string>&amp;Password</string>
            </attribute>
            <layout class="QGridLayout" >
              <property name="margin" stdset="0" >
                <number>11</number>
              </property>
              <property name="spacing" stdset="0" >
                <number>6</number>
              </property>
              <item row="2" column="0" >
                <widget class="QLabel" name="TextLabel2" >
                  <property name="text" stdset="0" >
                    <string>New password:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="0" column="1" >
                <widget class="QLineEdit" name="edtCurrent" >
                  <property name="echoMode" stdset="0" >
                    <enum>QLineEdit::Password</enum>
                  </property>
                </widget>
              </item>
              <item row="3" column="1" >
                <widget class="QLineEdit" name="edtPswd2" >
                  <property name="echoMode" stdset="0" >
                    <enum>QLineEdit::Password</enum>
                  </property>
                </widget>
              </item>
              <item row="3" column="0" >
                <widget class="QLabel" name="TextLabel3" >
                  <property name="text" stdset="0" >
                    <string>Retype new password:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="0" column="0" >
                <widget class="QLabel" name="TextLabel1" >
                  <property name="text" stdset="0" >
                    <string>Current password:</string>
                  </property>
                  <property name="alignment" stdset="0" >
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                </widget>
              </item>
              <item row="4" column="0" >
                <spacer name="Spacer6" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
              <item rowspan="1" row="1" column="0" colspan="2" >
                <widget class="Line" name="Line2_2" >
                  <property name="frameShape" stdset="0" >
                    <enum>QFrame::HLine</enum>
                  </property>
                  <property name="frameShadow" stdset="0" >
                    <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="frameShape" stdset="0" >
                    <enum>QFrame::HLine</enum>
                  </property>
                </widget>
              </item>
              <item row="2" column="1" >
                <widget class="QLineEdit" name="edtPswd1" >
                  <property name="echoMode" stdset="0" >
                    <enum>QLineEdit::Password</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <pixmapfunction></pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>QWidget</class>
      <extends></extends>
      <header location="global" >qwidget.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
      <properties>
        <property type="String" >text</property>
      </properties>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>tabWnd</tabstop>
    <tabstop>edtUin</tabstop>
    <tabstop>edtFirst</tabstop>
    <tabstop>edtLast</tabstop>
    <tabstop>edtNick</tabstop>
    <tabstop>cmbRandom</tabstop>
    <tabstop>cmbStatus</tabstop>
    <tabstop>edtOnline</tabstop>
    <tabstop>edtNA</tabstop>
    <tabstop>edtExtIP</tabstop>
    <tabstop>edtIntIP</tabstop>
    <tabstop>edtClient</tabstop>
    <tabstop>edtAutoReply</tabstop>
  </tabstops>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
</ui>
